packages/core/src/VNodes/AbstractNode.ts

```diff
@@ -13,17 +14,29 @@ export interface AbstractNodeParams {
 let id = 0;
 export abstract class AbstractNode extends EventMixin {
     readonly id = id;
+    _editable: boolean = null;
     /**
      * True if the node is editable.
+     * Can be overridden with a `Mode`.
      */
-    editable = true;
+    get editable(): boolean {
+        const attributes = this.modifiers.find(Attributes);
+        const editable = attributes?.get('contenteditable');
+        return typeof this._editable === 'boolean'
+            ? this._editable
+            : [undefined, 'true', ''].includes(editable);
+    }
+    set editable(state: boolean) {
+        this._editable = state;
+    }
```

Comment:
I don't think the logic should be in the node itself.
The node in the vdocument should not be aware of a contenteditable attribute that is only present in the DOM.
Another argument: It does not make sense for an `InlineNode`.

I would suggest that the mode `isEditable` would perform that logic.

I also suggest that the `mode.isPropertyX(node)` is always called anywhere in the code (which argue in favor of a mode being always present in the `JWEditor`).
And also make the code more modular, we could imagine a mode that do not care about "contenteditable" because it does know about any DOM concept.

The property of a node should be readonly as it describe how a node is defined but should not be manipulated to avoid confusion.

Reply:
I absolutely agree. You actually are the one who wrote this code :-D
But this is the part that I struggled with and finally gave up for the moment, hence the todo in the commit comment.

Reply:
Haha, it was a wip

Reply:
Is this still used then ? Removing this would make me happy. 🧸

Reply:
Sadly, no. I has evolved though.